run_pipeline.sh

```diff
@@ -31,7 +31,7 @@ ARCHIVE_FILE="$ARCHIVE_LOCATION/data-$RUN_ID.tar.gzip"
 ./docker-sync-engagement-db-to-coda.sh --incremental-cache-volume "$PIPELINE_NAME-engagement-db-to-coda-cache" "$USER" \
                         "$GOOGLE_CLOUD_CREDENTIALS_PATH" "$CONFIGURATION_MODULE" "$DATA_DIR"
 
-./docker-sync-coda-to-engagement-db.sh --incremental-cache-volume "$PIPELINE_NAME-coda-to-engagement-db-cache" "$USER" \
+./docker-sync-coda-to-engagement-db.sh --incremental-cache-volume -s "$PIPELINE_NAME-coda-to-engagement-db-cache" "$USER" \
```

Comment:
I think you should write the arg name out in full here instead of using -s, so it's clearer to people reading the code. (-s is fine for direct calls though)

Reply:
updated, thanks.